system-tests/projects/webpack-preprocessor-webpack-5/cypress.config.js

```diff
@@ -0,0 +1,50 @@
+// Since we are linking the `@cypress/webpack-preprocessor` package,
+// `require('webpack')` will resolve the version found in our repo (v4).
+// We need to override the webpack require to point to the local node_modules
+const Module = require('module')
+
+const originalModuleLoad = Module._load
+
+Module._load = function (request, parent, isMain) {
+  if (request === 'webpack' || request.startsWith('webpack/')) {
+    const resolvePath = require.resolve(request, {
```

Comment:
Do we need to try/catch this?

Reply:
Wouldn't hurt adding it for safety, where it falls back to the original load if it throws

Reply:
Ok, I'll leave it to you, dropped a ✅